app.js

```diff
@@ -8,8 +8,8 @@ var passport = require("passport");
 var dotenv = require("dotenv");
 var session = require("express-session");
 
-require('./util/passport')(passport);
 dotenv.config({path: './.env'});
+require('./util/passport')(passport);
```

Comment:
had to shift the dotenv config line above passport instantiation as the latter could not receive the variables from `.env` file